lib/xandra/cluster/pool.ex

```diff
@@ -152,25 +153,24 @@ defmodule Xandra.Cluster.Pool do
     queue_before_connecting_opts = Keyword.fetch!(cluster_opts, :queue_before_connecting)
     queue_before_connecting_timeout = Keyword.fetch!(queue_before_connecting_opts, :timeout)
 
-    data =
-      %__MODULE__{
-        pool_options: pool_opts,
-        contact_nodes: Keyword.fetch!(cluster_opts, :nodes),
-        load_balancing_module: lb_mod,
-        load_balancing_state: lb_mod.init(lb_opts),
-        autodiscovered_nodes_port: Keyword.fetch!(cluster_opts, :autodiscovered_nodes_port),
-        xandra_mod: Keyword.fetch!(cluster_opts, :xandra_module),
-        control_conn_mod: Keyword.fetch!(cluster_opts, :control_connection_module),
-        target_pools: Keyword.fetch!(cluster_opts, :target_pools),
-        name: Keyword.get(cluster_opts, :name),
-        pool_supervisor: pool_sup,
-        refresh_topology_interval: Keyword.fetch!(cluster_opts, :refresh_topology_interval),
-        reqs_before_connecting: %{
-          queue: :queue.new(),
-          max_size: Keyword.fetch!(queue_before_connecting_opts, :buffer_size)
-        },
-        sync_connect_ref: sync_connect_ref_or_nil && {parent, sync_connect_ref_or_nil}
-      }
+    data = %__MODULE__{
```

Comment:
Can we limit unnecessary changes to a minimum? I prefer your formatting too but let's do it in another PR, because I can't see if something changed here due to the big diff.

Reply:
Sorry, that'd be the work of `mix format`.

Reply:
Yeah, I thought so but I think you might be running it on a different version of Elixir or something, because CI doesn't really complain about files not being formatted...